resources/webviews/aksclusterproperties/clusterproperties.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>AKS Cluster Properties</title>
+    <!-- Link to the css file -->
+    <link rel="stylesheet" href="{{cssuri}}">
+    <script src="https://use.fontawesome.com/7dddd54b5c.js"></script>
+    <script src="https://code.jquery.com/jquery-3.5.1.min.js"
+        integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
+
+</head>
+
+<body>
+    <div class="heading">
+        <h2 class="content-header">AKS Cluster Properties of {{name}}</h2>
+    </div>
+
+    <div class="insight-container">
+        <div class="panel panel-default data-container">
+            <div class="panel-body data-container-body">
+
+                <div class="panel-body data-container-body allocatebody">
+                    <div class="panel-body">
+                        <table>
+                            <tbody>
+                                <tr>
+                                    <th>Properties</th>
+                                    <th>Values</th>
+                                </tr>
+
+                                <tr>
+                                    <td>Provisioning State</td>
+                                    <td>{{clusterData.provisioningState}}</td>
+                                </tr>
+                                <tr>
+                                    <td>FQDN</td>
+                                    <td>{{clusterData.fqdn}}</td>
+                                </tr>
+                                <tr>
+                                    <td>Kubernetes Version</td>
+                                    <td>{{clusterData.kubernetesVersion}}</td>
+                                </tr>
+                            </tbody>
+                        </table>
+                        <b>Agent Pool Properties</b>
+                        <table>
+                            <tbody>
+                                <tr>
+                                    <th>Properties</th>
+                                    <th>Values</th>
+                                </tr>
+                                {{#each clusterData.agentPoolProfiles}}
+                                <tr>
```

Comment:
I wonder if it'd be a good idea to move the `{{#each}}` block outside the `<table>` element. That way, if there's more than one agent pool you get a table for each one. The bold `Agent Pool Properties` could also contain {{this.name}}, so it's clear which is which. What do you reckon?